### ISSUE TITLE
Can't save navigation item without title property set

### ISSUE THREAD

**Issue by mzronek** [VirtusLab-Open-Source/strapi-plugin-navigation#92]
# Steps to reproduce

Local environment connected directly to a MySQL database.
custom.js for plugin configuration:

```
module.exports = ({ env }) => ({
  plugins: {
    navigation: {
      excludedContentTypes: ["plugins::", "strapi"],
      allowedLevels: 1,
      contentTypesNameFields: {
        page: ["title"],
      },
    },
  },
});
```

* Add new navigation item, set URL field and select collection and entry
* The title field needs to be left empty
* Press "Create Item" button
* Press "Save" button

![image](https://user-images.githubusercontent.com/3847700/130689043-2f5feff8-3750-44c7-b70e-16f56eb4d988.png)

Strapi shows an error: "An error occurred"
Terminal output shows the error:

```
[2021-08-24T18:43:49.559Z] error Error: insert into `navigations_items` (`created_at`, `menuAttached`, `order`, `path`, `type`, `uiRouterKey`, `updated_at`) values ('2021-08-24 18:43:49.516', false, 2, 'impressum', 'INTERNAL', 'impressum', '2021-08-24 18:43:49.516') - ER_NO_DEFAULT_FOR_FIELD: Field 'title' doesn't have a default value
    at Query.Sequence._packetToError (/workspace/backend/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/workspace/backend/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/workspace/backend/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/workspace/backend/node_modules/mysql/lib/protocol/Parser.js:433:10)
```

# Possible fix

The title property is omitted for the insert statement, but does not have a default value in the MySQL table column definition.

We could set a default in the model's settings so the title column can be null. Or we might pass the title as an empty string.

Thank you and kind regards,
mat

**Comment by sapkra** [comment]
![image](https://user-images.githubusercontent.com/8087568/168705577-32b8aa36-d64c-49d1-9dc8-b519a65fe76d.png)

Confirmed when trying to edit a nav item. Initially saving it works fine though.

**Comment by ltsNotMike** [comment]
Hi @sapkra, can you please provide version of node, strapi and navigation plugin that you are using.

**Comment by sapkra** [comment]
@ltsNotMike 

strapi-plugin-navigation: 2.1.0-beta.1
node: 16.15.0
strapi: 4.1.10

**Comment by cyp3rius** [comment]
@sapkra fix released as part of major version `2.1.0`
